src/CMakeLists.txt

```diff
@@ -534,3 +534,11 @@ ilink(${CMAKE_INSTALL_PREFIX}/${SWIPL_INSTALL_ARCH_EXE}/swipl-ld
 endif()
 
 endif(WIN32)
+
+# Create symbolic link to lib directory, since rpath is broken
+# in many android versions
+if(ANDROID)
```

Comment:
I guess ideally we have an option to install libswipl in `${CMAKE_INSTALL_PREFIX}/lib` _instead_ of in the local tree and forget about the RPATH (I think that is automatic)  We could than activate this for Android, but the Debian distributions also prefer installations without `rpath`.   Is that a plan?

Reply:
Good, should it be caled SWIPL_INSTALL_IN_LIB?